crates/types/src/logs/metadata.rs

```diff
@@ -220,7 +220,8 @@ impl From<ProviderConfiguration> for crate::protobuf::cluster::BifrostProvider {
                 result.provider = ProviderKind::Replicated.to_string();
                 result.replication_property = Some(cluster::ReplicationProperty {
                     replication_property: config.replication_property.to_string(),
-                })
+                });
+                result.target_nodeset_size = config.target_nodeset_size.as_u32();
```

Comment:
So we actually change the value but when calling `restatectl config get` we don't report it back, right?

Reply:
yes, that's exactly what happens. you can even verify that by doing `restatectl metadata get -k bifrost_config`